app/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:requestLegacyExternalStorage="true">
```

Comment:
I was receiving `Is your SD card read-only?` errors during taking snapshots. This line is supposed to fix it based on comments on link on line 7. But also this file should exists in androidTest dir.

Reply:
If you were recording snapshots on API 24, you should have not seen this error